third_party/move/move-vm/integration-tests/src/tests/mod.rs

```diff
@@ -94,7 +94,7 @@ fn execute_function_for_test(
     MoveVM::execute_loaded_function(
         func,
         args,
-        &mut TransactionDataCache::empty(),
+        &mut TransactionDataCache::empty(true),
```

Comment:
What's our usual approach when rolling out flags @georgemitenkov? Is there a way to test both behaviors, or would it be better to enable in tests at the same time as when the flag itself is enabled?